kiwi/storage/subformat/base.py

```diff
@@ -80,6 +82,24 @@ def post_init(self, custom_args):
         """
         pass
 
+    def has_raw_disk(self):
+        return os.path.exists(self.diskname)
+
+    def resize_raw_disk(self, size_bytes):
+        """
+        Resize raw disk image to specified size
+        """
+        current_byte_size = os.path.getsize(self.diskname)
+        size_bytes = int(size_bytes)
+        if size_bytes <= current_byte_size:
```

Comment:
I am not sure the equality is strictly needed here. If the raw disk has the same size as requested I think it should just skip this step, but no fail. Simple example that happened to me while testing it: I forgot to include the type and profile I used to build the image (`image="oem"` and `profile="vmxFlavour"`), so the command resize selected the default ones (`image="iso"` and `profile="vmxFlavour"`), so it resized the raw disk but it did not perform any format process because in iso type there is no format specified (I did the previous oem build with `format="qcow2"`). The I run again the same command including the oem type expecting it to not do any resize (or do a transparent/dummy resize) and also include the image formatting, however it failed with this KiwiResizeRawDiskError. In order to get the formatted qcow2 disk I had to resize again to a bigger size than I initially wanted.

Maybe if equal size is requested the command could be skipped, or even run it since running a resize process with `qemu-img` twice (with the same size value) over the same file does not change the initial result (I validated it with diff)